pyteal/ast/abi/string.py

```diff
@@ -28,10 +46,45 @@ def type_spec(self) -> StringTypeSpec:
         return StringTypeSpec()
 
     def get(self) -> Expr:
-        return substringForDecoding(
-            self.stored_value.load(),
-            startIndex=Int(Uint16TypeSpec().byte_length_static()),
+        return Suffix(
+            self.stored_value.load(), Int(Uint16TypeSpec().byte_length_static())
         )
 
+    def set(
+        self,
+        value: Union[
+            Sequence[T],
+            DynamicArray[T],
+            ComputedValue[DynamicArray[T]],
+            "String",
+            str,
+            Expr,
```

Comment:
same here: do we have the implementation supporting all these types? For `Sequence[T]` and `DynamicArray[T]`, support seems lacking.